backend/development.ini.sample

```diff
@@ -357,15 +357,15 @@ email.reply.activated = False
 ### IMAP ###
 # Tracim email reply feature will read all mails in following IMAP,
 # to be able to transform email responses into Tracim app comments.
-email.reply.imap.server = your_imap_server
-email.reply.imap.port = 993
-email.reply.imap.user = your_imap_user
-email.reply.imap.password = your_imap_password
+; email.reply.imap.server = your_imap_server
+; email.reply.imap.port = 993
+; email.reply.imap.user = your_imap_user
+; email.reply.imap.password = your_imap_password
 email.reply.imap.folder = INBOX
 email.reply.imap.use_ssl = True
 # IMAP IDLE feature is for real-time notifications (its not necessary to use function send & receive)
 # if this option is available in your imap server, change this parameter to True
-email.reply.imap.use_idle = False
+email.reply.imap.use_idle = True
```

Comment:
I don't understand why this parameter is now `True` by default because we not know `if this option is available in user imap server`.

Reply:
You're right.

Reply:
we need to set default value to false

current behaviour is false in config and true as default value, we should comment config with false and set default value as false.

Reply:
> we should comment config with false and set default value as false.

Agreed. It is done ?

Reply:
yes.